Steer/DigitizerWorkflow/src/SimpleDigitizerWorkflow.cxx

```diff
@@ -489,7 +493,14 @@ WorkflowSpec defineDataProcessing(ConfigContext const& configcontext)
     // connect the FIT digit writer
     specs.emplace_back(o2::ft0::getFT0DigitWriterSpec(mctruth));
   }
-
+  // the CTP part
```

Comment:
move this after other digitization devices and instead of passing `mctruth`, pass the `detList` to see what triggerInputs you can subscribe to.